package.json

```diff
@@ -25,15 +25,13 @@
   },
   "devDependencies": {
     "@nightwatch/vue": "^3.1.2",
-    "@tsconfig/node20": "^20.1.4",
     "@types/jsdom": "^21.1.7",
     "@types/node": "^20.16.11",
     "@vitejs/plugin-vue": "^5.1.4",
     "@vitest/eslint-plugin": "1.1.7",
     "@vue/eslint-config-prettier": "^10.0.0",
     "@vue/eslint-config-typescript": "^14.0.1",
     "@vue/test-utils": "^2.4.6",
-    "@vue/tsconfig": "^0.5.1",
```

Comment:
this guy has "resolveJsonModule": true, (we use json files for translations)

Reply:
:+1:  I've added it to tsconfig.app.json